File: ElaroMobil/app/src/main/java/com/calisma/e_ticaretsitesison/models/Urun.kt
package com.calisma.e_ticaretsitesison.models

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import com.google.gson.annotations.SerializedName

@Parcelize
data class Urun(
    @SerializedName("urunID")
    val urunID: Int = 0,

    @SerializedName("ad")
    val ad: String? = null,

    @SerializedName("aciklama")
    val aciklama: String? = null,

    @SerializedName("fiyat")
    val fiyat: Double? = null,

    @SerializedName("stokAdedi")
    val stokAdedi: Int? = null,

    @SerializedName("kategoriID")
    val kategoriID: Int? = null,

    @SerializedName("marka")
    val marka: String? = null,

    @SerializedName("renk")
    val renk: String? = null,

    @SerializedName("gorselURL")
    val gorselURL: String? = null
) : Parcelable

File: ElaroAPI/ElaroApi/Models/Urun.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ElaroApi.Models
{
    [Table("Urun2")]
    public class Urun
    {
        [Key]
        public int UrunID { get; set; }

        [Column("Adi")]
        public string Ad { get; set; }

        [Column("Aciklama")]
        public string Aciklama { get; set; }

        [Column("Fiyat")]
        public decimal Fiyat { get; set; }

        [Column("StokAdedi")]
        public int StokAdedi { get; set; }

        [Column("KategoriID")]
        public int KategoriID { get; set; }

        [Column("Marka")]
        public string Marka { get; set; }

        [Column("Renk")]
        public string Renk { get; set; }

        [Column("GorselURL")]
        public string GorselURL { get; set; }
    }
}
